app/components/notification_message_component.rb

```diff
@@ -0,0 +1,40 @@
+class NotificationMessageComponent < ViewComponent::Base
+  include ViewHelper
+
+  def initialize(notification_type, message:, secondary_message: nil, message_link: nil)
```

Comment:
Should it be notification_type: here in order to be consistent with the other arguments?

Reply:
I think it's not necessary and it reads better this way.
I used the hash format for the other attributes just so it's clear what their purpose is.